Travis tests failing with pytest 4.0.0

The BOTO_CONFIG=/tmp/nowhere test run is failing on master (e.g., see [job 455625370]).  I haven't looked at it in detail, but the failures seem to be related to direct fixture calls being unsupported in the latest pytest (v4.0.0).  For example

```

    def test_distribution_sub():
>       (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages()
E       _pytest.warning_types.RemovedInPytest4Warning: Fixture "setup_packages" called directly. Fixtures are not meant to be called directly, are created automatically when test functions request them as parameters. See https://docs.pytest.org/en/latest/fixture.html for more information.

niceman/distributions/tests/test_redhat.py:159: RemovedInPytest4Warning
```

[job 455625370]: https://travis-ci.org/ReproNim/niceman/jobs/455625370

--- a/niceman/distributions/tests/test_debian.py
+++ b/niceman/distributions/tests/test_debian.py
@@ -227,8 +227,8 @@ def test_package_satisfies(setup_packages):
     assert p1v11ai.satisfies(p1v11)
 
 @pytest.fixture
-def setup_distributions():
-    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages()
+def setup_distributions(setup_packages):
+    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages
     d1 = DebianDistribution(name='debian 1')
     d1.packages = [p1]
     d2 = DebianDistribution(name='debian 2')
@@ -249,8 +249,8 @@ def test_distribution_statisfies(setup_distributions):
     assert not d1.satisfies(d2)
     assert d2.satisfies(d1)
 
-def test_distribution_sub():
-    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages()
+def test_distribution_sub(setup_packages):
+    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages
     d1 = DebianDistribution(name='debian 1')
     d1.packages = [p1, p2]
     d2 = DebianDistribution(name='debian 2')

--- a/niceman/distributions/tests/test_redhat.py
+++ b/niceman/distributions/tests/test_redhat.py
@@ -112,8 +112,8 @@ def setup_packages():
 
 
 @pytest.fixture
-def setup_distributions():
-    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages()
+def setup_distributions(setup_packages):
+    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages
     d1 = RedhatDistribution(name='debian 1')
     d1.packages = [p1]
     d2 = RedhatDistribution(name='debian 2')
@@ -155,8 +155,8 @@ def test_distribution_statisfies(setup_distributions):
     assert d2.satisfies(d1)
 
 
-def test_distribution_sub():
-    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages()
+def test_distribution_sub(setup_packages):
+    (p1, p1v10, p1v11, p1ai, p1aa, p1v11ai, p2) = setup_packages
     d1 = RedhatDistribution(name='debian 1')
     d1.packages = [p1, p2]
     d2 = RedhatDistribution(name='debian 2')

